<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Chart</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2018</year>
      <month>7</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2018</year>
      <month>7</month>
      <day>15</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Hours</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Application Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>23</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Api Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>a6297ab5c9c2459c8656f532772b5f49</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
